scripts/hubot-sumologic-image-widget/main.coffee

```diff
@@ -0,0 +1,47 @@
+# Payou Experimental lab for display sumologic widget as
+# a screenshot image
+
+
+# Idea #1
+# Input - Widget data from sumologic
+# 1) Use combinaison of jsdom and highchart to generate an svg graph
+# 2) Use svg2img to generate image file
+# 3) Render it in an accessible place but temporary
```

Comment:
So cool :)
S3 was for public purpose (if you want share the link to other people we don't have access to slack)